lib/ain-evm/src/backend.rs

```diff
@@ -187,6 +187,68 @@ impl EVMBackend {
             .map(|acc| acc.nonce)
             .unwrap_or_default()
     }
+
+    pub fn get_account_storage(&self, contract: H160, storage_index: H256) -> Result<U256> {
+        let account = self.get_account(&contract).unwrap_or(Account {
+            nonce: U256::zero(),
+            balance: U256::zero(),
+            storage_root: H256::zero(),
+            code_hash: H256::zero(),
+        });
```

Comment:
Use let else { return U256::default() } to return early

Reply:
Fixed: 49ec7c6bd99f38f299817819353c1a652bdc55a4